peer.go

```diff
@@ -553,7 +555,7 @@ func (p *peer) pushBlockMsg(sha *wire.ShaHash, doneChan, waitChan chan struct{})
 	// We only send the channel for this message if we aren't sending
 	// an inv straight after.
 	var dc chan struct{}
-	sendInv := p.continueHash != nil && p.continueHash.IsEqual(sha)
+	sendInv := p.ContinueHash() != nil && p.ContinueHash().IsEqual(sha)
```

Comment:
Seems like the continue hash is a feature of the handling code and thus shouldn't be defined on the peer.